sdk/template/ci.yml

```diff
@@ -7,7 +7,9 @@ trigger:
     - release/*
   paths:
     include:
-    - sdk/template/
+    - sdk/template/ 
+    # eng/common code changes trigger template pipeline for basic checking.
```

Comment:
One thing we might get hit by here is that I believe we copy this ci.yml file as part of creating new service pipelines and if we do that we don't want them to be triggered on eng/common changes. @chunyu3 @annelo-msft do we use this file in our sdk template generation?

Reply:
It's one of the ones. I created a `dotnet new` template for it and could remove it from there, but people who just copy the directory would remove it.

I would add a comment above this to remove the line at the very least. It something else we should track in #27397.

Reply:
yes, We will copy this ci.yml file to the new generated service package. and the path `- /sdk/template` will be update to `- /sdk/<service>`. After discuss with @heaths , we will update this ci.yml file, that for normal service, we will not include `-eng/common` in ci.yml, just for the sample/template RP, it will include `- eng/common` which is used to test if the template is correct or not.